fix videowidget cursor handling
videowidget.cpp explicitly sets Qt::ArrowCursor under certain
circumstances; after that it is impossible to hide the cursor over the
video widget (needed e.g. in fullscreen mode)
backport of 950148

http://websvn.kde.org/?view=rev&revision=950739
--- a/xine/videowidget.cpp
+++ b/xine/videowidget.cpp
@@ -472,10 +472,6 @@
 {
     K_XT(VideoWidget);
 
-    if (cursor().shape() == Qt::BlankCursor) {
-        setCursor(QCursor(Qt::ArrowCursor));
-    }
-
     x11_rectangle_t   rect;
     xine_event_t      *event = new xine_event_t;
     xine_input_data_t *input = new xine_input_data_t;
